css/styles.css

```diff
@@ -56,6 +56,88 @@ a {
   display: none;
 }
 
+.popup {
+  position: fixed;
+  top: 0;
+  width: 100vw;
+  height: 100vh;
+  background-color: white;
+  border: 20px solid #c1c7d0;
+}
+
+.xsign {
+  position: absolute;
+  top: 0;
+  margin-left: 298px;
+  padding-top: 50px;
+  color: #fff;
+  z-index: 1;
+}
+
+.pop-img {
+  width: 370px;
+  position: relative;
+  height: 320px;
+  padding-right: 35px;
+}
+
+.phead {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  width: 260px;
+  margin-left: 10px;
+}
+
+.ptext {
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.plist {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  list-style: none;
+  padding-left: 0;
+}
+
+.panc {
+  padding: 8px 12px;
+  background: #ebf0ee;
+  color: #3a4a42;
+  text-align: center;
+  text-decoration: none;
+  border-radius: 10px;
+  margin-right: 10px;
+}
+
+.pbuttons {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  list-style: none;
+  gap: 47px;
+}
```

Comment:
- [ ] Dear @danielmatama, As the previous code reviewer mentioned, I noticed that the two buttons are hidden for the mobile version `375px` screen, kindly adjust this for a better user experience. 🙏